runtime/_data.ts

```diff
@@ -46,11 +46,6 @@ export const sidebar = [
       {
         label: "Deno CLI",
         items: [
-          "/runtime/reference/cli/unstable_flags/",
-          {
-            label: "Environment Variables",
-            id: "/runtime/reference/cli/env_variables/",
```

Comment:
Will this page be completely removed from the sidebar?

Reply:
yes

Reply:
it is covered in the same named page that is in references